include/net/lwm2m.h

```diff
@@ -267,6 +267,93 @@ int lwm2m_device_set_pwrsrc_current_ma(int index, int current_ma);
  */
 int lwm2m_device_add_err(u8_t error_code);
 
+#if defined(CONFIG_LWM2M_CONN_MON_OBJ_SUPPORT)
+/**
+ * @brief Register an available net bearer with the LwM2M Connectivity
+ *        Monitoring object.
+ *
+ * @param[in] bearer_id Network Bearer ID.
```

Comment:
What exactly is the bearer_id value, is this related to network interface or something that exists already in the network stack?

Reply:
This was removed in the latest patch series as it is now up to the user to set these resources via generic methods.  But here's the description of what a "bearer" is according to the Connectivity Monitoring spec:
```
Indicates the network bearer used for the current LwM2M communication session
from the network bearer list below. 
 The number range is split into three categories:
 0 - 20 are Cellular Bearers 
 21 - 40 are Wireless Bearers 
 41 - 50 are Wireline Bearers
 
 More specifically: 
 0: GSM cellular network
 1: TD-SCDMA cellular network
 2: WCDMA cellular network
 3: CDMA2000 cellular network
 4: WiMAX cellular network
 5: LTE-TDD cellular network
 6: LTE-FDD cellular network
 7: NB-IoT
 8 - 20: Reserved for other types of cellular network
 21: WLAN network
 22: Bluetooth network
 23: IEEE 802.15.4 network
 24 - 40: Reserved for other types of local wireless network
 41: Ethernet
 42: DSL
 43: PLC
 44 - 50: reserved for other types of wireline networks.
```